src/pages/TransactionIndexer/Title.tsx

```diff
@@ -0,0 +1,18 @@
+import {Stack, Typography} from "@mui/material";
+import TitleHashButton, {HashType} from "../../components/TitleHashButton";
+import {TransactionType} from "../../components/TransactionType";
+import {Transactions} from "../../gql/graphql";
+
+type TransactionTitleProps = {
+  transaction: Transactions;
+};
+
+export default function TransactionTitle({transaction}: TransactionTitleProps) {
+  return (
+    <Stack direction="column" spacing={2} marginX={1}>
+      <Typography variant="h3">Transaction</Typography>
+      <TitleHashButton hash={transaction.hash} type={HashType.TRANSACTION} />
+      <TransactionType type={transaction.transaction_type} />
+    </Stack>
+  );
+}
```

Comment:
For example, this has like no change from the non-indexer version of this component, but we made a whole new file :(